mingw-w64-git-lfs/PKGBUILD

```diff
@@ -40,7 +40,8 @@ source=("https://github.com/github/git-lfs/releases/download/v$pkgver/$zipname"
 # Git LFS' source .zip now contains symbolic links that bsdtar does not like...
 noextract=("${src_zip_url##*/}")
 
-sha256sums=("$sha256sum" SKIP)
+sha256sums=('1df5874f22c35c679159f0aaf9e24333051f52768eade0204d22200b79141743'
```

Comment:
Defining one checksum for three diffrent files is probably not going to mork.

Reply:
I think we'll need to avoid [this `updpkgsums`](https://github.com/git-for-windows/git-for-windows-automation/blob/main/.github/workflows/open-pr.yml#L155) call for Git LFS and somehow query the checksums from GitHub [like `please.sh` did](https://github.com/git-for-windows/build-extra/blob/main/please.sh#L2634-L2669)

Reply:
Indeed...